src/components/Clickable/Clickable.tsx

```diff
@@ -1,13 +1,46 @@
 import React from 'react';
 import classNames from 'classnames';
+import { Link, LinkProps } from 'react-router-dom';
+
 import './Clickable.scss';
 
-export type ClickableProps = React.ButtonHTMLAttributes<HTMLButtonElement>;
+type ButtonElProps = React.ButtonHTMLAttributes<HTMLButtonElement>;
+type AnchorElProps = React.AnchorHTMLAttributes<HTMLAnchorElement>;
+
+export type ClickableProps<T = ButtonElProps | AnchorElProps | LinkProps> = T & {
```

Comment:
I don't this this has to be generic, just do `(ButtonElProps | AnchorElProps | LinkProps) & {...}`

Reply:
it doesn't work because it always requires that `to` from `LinkProps`; 

if not, it breaks while comparing subtypes of `ButtonElProps` and `AnchorElProps`

Reply:
I think you don't have to make the type generic to fix that, just export all the subtypes and use the `ButtonProps` one for `ColorSwatch`.
Something like:
![image](https://user-images.githubusercontent.com/18576413/82573047-f321e400-9b8d-11ea-83cb-41a55ccba9be.png)
![image](https://user-images.githubusercontent.com/18576413/82573098-02a12d00-9b8e-11ea-9128-f3820f30bfc5.png)